src/app/dashboard/actions.js

```diff
@@ -27,7 +27,7 @@ const updateLastRunTimestamp = `UPDATE wip_configurations.spark_streaming_table_
    SET last_run_timestamp = '' WHERE source_system_name = ? and source_table_name = ? `;
 
 const deleteRowQuery = `UPDATE wip_configurations.spark_streaming_table_config 
-  SET voided_by = ?, active = ?, updated_date = ?
+  SET voided_by = ?, active = ?, updated_date = ?, update_by = ?
```

Comment:
Add updated_by for `updateRowQuery, insertRowQuery, deleteRowQuery and unvoidRowQuery`